ci-operator/step-registry/openshift/microshift/e2e/openshift-conformance/openshift-microshift-e2e-openshift-conformance-commands.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+set -euo pipefail
+
+trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wait; fi' TERM
+
+INSTANCE_PREFIX="${NAMESPACE}"-"${JOB_NAME_HASH}"
+GOOGLE_PROJECT_ID="$(< ${CLUSTER_PROFILE_DIR}/openshift_gcp_project)"
+GOOGLE_COMPUTE_REGION="${LEASED_RESOURCE}"
+GOOGLE_COMPUTE_ZONE="$(< ${SHARED_DIR}/openshift_gcp_compute_zone)"
+if [[ -z "${GOOGLE_COMPUTE_ZONE}" ]]; then
+  echo "Expected \${SHARED_DIR}/openshift_gcp_compute_zone to contain the GCP zone"
+  exit 1
+fi
+
+mkdir -p "${HOME}"/.ssh
+
+mock-nss.sh
+
+# gcloud compute will use this key rather than create a new one
+cp "${CLUSTER_PROFILE_DIR}"/ssh-privatekey "${HOME}"/.ssh/google_compute_engine
+chmod 0600 "${HOME}"/.ssh/google_compute_engine
+cp "${CLUSTER_PROFILE_DIR}"/ssh-publickey "${HOME}"/.ssh/google_compute_engine.pub
+echo 'ServerAliveInterval 30' | tee -a "${HOME}"/.ssh/config
+echo 'ServerAliveCountMax 1200' | tee -a "${HOME}"/.ssh/config
+chmod 0600 "${HOME}"/.ssh/config
+
+# Copy pull secret to user home
+cp "${CLUSTER_PROFILE_DIR}"/pull-secret "${HOME}"/pull-secret
+
+gcloud auth activate-service-account --quiet --key-file "${CLUSTER_PROFILE_DIR}"/gce.json
+gcloud --quiet config set project "${GOOGLE_PROJECT_ID}"
+gcloud --quiet config set compute/zone "${GOOGLE_COMPUTE_ZONE}"
+gcloud --quiet config set compute/region "${GOOGLE_COMPUTE_REGION}"
+
+# Run openshift/conformance test suite
+# Later on a list of skipped tests can be provided and filtered out
+# from the suite using combination of "openshift-tests run openshift/conformance --dry-run | grep | openshift-tests run -f"
+cat  > "${HOME}"/run-test.sh <<'EOF'
+export KUBECONFIG=/var/lib/microshift/resources/kubeadmin/kubeconfig
+openshift-tests run openshift/conformance \
```

Comment:
Can we change this a bit? We will not be running all tests and we need some way of skipping them at will. Something along the lines of single node: https://github.com/openshift/release/blob/master/ci-operator/step-registry/single-node/e2e/test/single-node-e2e-test-commands.sh#L264-L281

Reply:
Absolutely. I am testing the wiring now. Would be great to introduce TEST_SKIPS, TEST_SUITE, TEST_ARGS and other envs as well for better customization.